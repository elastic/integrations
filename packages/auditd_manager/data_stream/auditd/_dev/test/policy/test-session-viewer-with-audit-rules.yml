data_stream:
  vars:
    session_data: true
    session_data_processors: "  - add_session_metadata:\n     backend: \"auto\""
    audit_rules: "# Unauthorized access attempts to files (unsuccessful).\n-a always,exit -F arch=b32 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EACCES -F auid>=1000 -F auid!=4294967295 -F key=access\n-a always,exit -F arch=b32 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EPERM -F auid>=1000 -F auid!=4294967295 -F key=access\n-a always,exit -F arch=b64 -S open,truncate,ftruncate,creat,openat,open_by_handle_at -F exit=-EACCES -F auid>=1000 -F auid!=4294967295 -F key=access\n-a always,exit -F arch=b64 -S open,truncate,ftruncate,creat,openat,open_by_handle_at -F exit=-EPERM -F auid>=1000 -F auid!=4294967295 -F key=access"
