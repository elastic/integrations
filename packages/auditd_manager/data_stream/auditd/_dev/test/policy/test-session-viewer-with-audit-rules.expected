inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: auditd_manager
      name: test-session-viewer-with-audit-rules-auditd_manager
      streams:
        - audit_rules: |-
            # Session data audit rules
            -a always,exit -F arch=b64 -S execve,execveat -k exec
            -a always,exit -F arch=b64 -S exit_group
            -a always,exit -F arch=b64 -S setsid
            # Unauthorized access attempts to files (unsuccessful).
            -a always,exit -F arch=b32 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EACCES -F auid>=1000 -F auid!=4294967295 -F key=access
            -a always,exit -F arch=b32 -S open,creat,truncate,ftruncate,openat,open_by_handle_at -F exit=-EPERM -F auid>=1000 -F auid!=4294967295 -F key=access
            -a always,exit -F arch=b64 -S open,truncate,ftruncate,creat,openat,open_by_handle_at -F exit=-EACCES -F auid>=1000 -F auid!=4294967295 -F key=access
            -a always,exit -F arch=b64 -S open,truncate,ftruncate,creat,openat,open_by_handle_at -F exit=-EPERM -F auid>=1000 -F auid!=4294967295 -F key=access
          backlog_limit: 8192
          backpressure_strategy: auto
          condition: ${host.platform} == 'linux'
          data_stream:
            dataset: auditd_manager.auditd
            type: logs
          failure_mode: silent
          immutable: false
          include_raw_message: true
          include_warnings: false
          processors:
            - add_session_metadata:
                backend: auto
          rate_limit: 0
          resolve_ids: true
          socket_type: ""
          tags:
            - auditd_manager-auditd
          type: audit/auditd
      type: audit/auditd
      use_output: default
namespaces: []
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - logs-auditd_manager.auditd-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references: []
